Gemfile

```diff
@@ -17,7 +17,7 @@ group :test do
   gem 'rspec'
   gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.9.0')
   if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils', '~> 0.11'
+    gem 'sqa-utils', '0.12.1'
```

Comment:
@cprice404 Actually, all the other branches are using `0.13`.

Notice that in this patch `~> 0.11` (less restrictive) is being replaced with `0.12.1`. Last week when I released `0.13.0` of the `sqa-utils` gem I had to submit a patch to puppet-server stable branch because the version of `beaker` being used was incompatible with `sqa-utils 0.13.0' which probably means `0.13.0` should have been a major version increase, but it's too late for that and (hopefully) in the near future we will be deprecating `sqa-utils` in favor of a more aptly-named `beaker-genconfig` gem.

While pinning to `0.12.1` in puppet-server master and 2.1.x branches won't cause any problems, I don't see any particular reason to do that given that these branches have been implicitly running off `0.13.0` since last week.